src/components/Auth/Auth.tsx

```diff
@@ -25,6 +25,7 @@ const AuthCapture: FunctionComponent<Props> = (props) => {
   const [, { findStep }] = useSdkOptions()
   const { translate } = useLocales()
   const retries = findStep('auth')?.options?.retries || 3
+  const apiUrl = getUrlsFromJWT(props.token).onfido_api_url
```

Comment:
The URLs should be already available here from the Redux store, are they not?

Reply:
right, makes sense. pushed another commit let me know if that makes more sense